yamllint/config.py

```diff
@@ -153,8 +153,29 @@ def validate_rule_conf(rule, conf):
         return False
 
     if isinstance(conf, dict):
-        if ('ignore' in conf and
-                not isinstance(conf['ignore'], pathspec.pathspec.PathSpec)):
+        if 'ignore-from-file' in conf and isinstance(
+            conf['ignore-from-file'], (str, list)
+        ):
+            if isinstance(conf['ignore-from-file'], str):
+                conf['ignore-from-file'] = [conf['ignore-from-file']]
+            if not (
+                isinstance(conf['ignore-from-file'], list)
+                and all(
+                    isinstance(line, str) and os.path.isfile(line)
+                    for line in conf['ignore-from-file']
+                )
+            ):
+                raise YamlLintConfigError(
+                    'invalid config: ignore-from-file should contain '
+                    'valid filename(s), either as a list or string.'
+                )
+            with fileinput.input(conf['ignore-from-file']) as ignore_file:
+                conf['ignore'] = pathspec.PathSpec.from_lines(
+                    'gitwildmatch', ignore_file
+                )
```

Comment:
Same kind of remark, for [consistency](https://github.com/adrienverge/yamllint/commit/2f8ad7003a8571e0c935bc6ba7fef04ed6f3fde6#diff-443dfa037d6d01528b63511e2301a7bbb7d2ef8d3dc85b1df3e7fabd8b0d52f5R112-R113) and easier maintenance/debugging/code evolution: can you use `f` instead of `ignore_file`?

Reply:
> can you use f instead of ignore_file?

Yes, fixed that.